app/controllers/application_controller.rb

```diff
@@ -7,12 +7,6 @@ class ApplicationController < ActionController::Base
 
   rescue_from ActionView::MissingTemplate, with: :routing_error
 
-  def require_http_auth
-    authenticate_or_request_with_http_basic do |username, password|
-      username == ApplicationConfig["APP_NAME"] && password == ApplicationConfig["APP_PASSWORD"]
-    end
-  end
-
```

Comment:
removed this as we don't use HTTP basic auth, we never call `require_http_auth` and we don't have those two env variables anywhere

Reply:
This is super legacy stuff 😆